drush.php

```diff
@@ -63,6 +73,6 @@
 // Preflight and run
 $preflight = new Preflight($environment);
 $runtime = new Runtime($preflight);
-$status_code = $runtime->run($_SERVER['argv']);
+$status_code = $runtime->run($argv);
```

Comment:
I think $argv should be assigned `$_SERVER['argv']` in the non-php-fpm code path.

Reply:
both are setted to the same, by decoding args (line 58 to 65).
you'd rather using $_SERVER['argv'] or $argv  line 76 ?